Concepts/concept.md

```diff
@@ -0,0 +1,504 @@
+
+* The bottom line is, if you need a medical word within your electronic records system, it needs to be defined within the 
+concept dictionary.  More detail about all the possible concepts in a later section.
+  
+## Sub Resource types
+
+### Concept Attribute.
+
+* If you wish to record extra information about concept, you can create Concept Attributes and assign them to Concept Types.
+
+* Concept attributes exists specifically to allow implementations to extend the data model.
+
+### Concept Name.
+
+* ConceptNames are the words or phrases used to express the idea of a Concept within a particular locale. 
+Because there may be many names for any concept.
+
+### Concept Mapping.
+
+* Concept Mappings are added to facilitate managing concept dictionaries and point to other concepts which have the same meaning. 
+Mappings are useful when you need to receive or send information to external systems, letting you define how your system's
+ concepts relate to external concepts such as standardized medical vocabularies (e.g., ICD, LOINC, SNOMED).
+
+* For example, add a mapping to a concept in the MCL dictionary. You can save the concept now and create some answers.
+
+### Concept Description.
+
+* A clear and concise description of the concept, as agreed upon by the members of the organization or the most commonly 
+referenced source.  
+
+## Available operations. 
+1.  [List concepts](#list-concepts)
+2.  [Create a concept](#create-a-concept)
+3.  [Update a concept](#update-a-concept)
+4.  [Delete a concept](#delete-a-concept)
+5.  [List concept mapping sub resource](#list-concept-mapping-sub-resources)
+6.  [Create concept mapping sub resource with properties](#create-a-concept-mapping-sub-resource-with-properties)
+7.  [Update concept mapping sub resource](#update-a-concept-mapping-sub-resource)
+8.  [Delete concept mapping sub resource](#delete-a-concept-mapping-sub-resource)
+9.  [List concept name sub resource](#list-concept-name-sub-resources)
+10. [Create concept name sub resource with properties](#create-a-concept-name-sub-resource-with-properties)
+11. [Update concept name sub resource](#update-concept-name-sub-resource)
+12. [Delete concept name sub resource](#delete-concept-name-sub-resource)
+13. [List concept attribute sub resource](#list-concept-attribute-sub-resources)
+14. [Create concept attribute sub resource with properties](#create-a-concept-attribute-sub-resource-with-properties)
+15. [Update concept attribute sub resource](#update-concept-attribute-sub-resource)
+16. [Delete concept attribute sub resource](#delete-concept-attribute-sub-resource)
+13. [List concept description sub resource](#list-concept-description-sub-resources)
+14. [Create concept description sub resource with properties](#create-a-concept-description-sub-resource-with-properties)
+15. [Update concept description sub resource](#update-concept-description-sub-resource)
+16. [Delete concept description sub resource](#delete-concept-description-sub-resource)
```

Comment:
Please remove the "sub resource" terms here. While we have more often used the term without a space (i.e., "subresource" instead of "sub resource"), I would remove it in lists like this, since it can make the documentation more confusing/intimidating. Better to have something like:
* List concept mappings
* Create concept mappings
* Update concept mappings
* Delete concept mappings
* List concept names
* etc.